src/components/change-requests/change-request-details/change-request-details/change-request-details.tsx

```diff
@@ -55,26 +54,24 @@ interface ChangeRequestDetailsProps {
 const ChangeRequestDetails: React.FC<ChangeRequestDetailsProps> = ({
   changeRequest
 }: ChangeRequestDetailsProps) => {
-  const reviewBtns = (
-    <div className={styles.btnsContainer}>
-      <ActionButton
-        link={`/change-requests/${changeRequest.crId}/accept`}
-        icon={faThumbsUp}
-        text="Accept"
-      />
-      <ActionButton
-        link={`/change-requests/${changeRequest.crId}/deny`}
-        icon={faThumbsDown}
-        text="Deny"
-      />
-    </div>
+  const reviewDropdown = (
+    <DropdownButton id="review-dropdown" title="Review">
+      <Dropdown.Item as={Link} to={`/change-requests/${changeRequest.crId}/accept`}>Accept</Dropdown.Item>
+      <Dropdown.Item as={Link} to={`/change-requests/${changeRequest.crId}/deny`}>Deny</Dropdown.Item>
+    </DropdownButton>
+  );
+
+  const implementCrDropdown = (
+    <DropdownButton id="implement-cr-dropdown" title="Implement Change Request">
+      <Dropdown.Item as={Link} to="/projects/new">Create New Project</Dropdown.Item>
+    </DropdownButton>
   );
 
   return (
     <>
       <PageTitle
         title={`Change Request #${changeRequest.crId}`}
-        actionButton={changeRequest.accepted !== undefined ? <></> : reviewBtns}
+        actionButton={changeRequest.accepted !== undefined ? (changeRequest.accepted! ? implementCrDropdown : <></>) : reviewDropdown}
```

Comment:
rather than using nested inline ifs, I would recommend doing something like

```ts
let btn = <></>;
if (condition1) btn = X;
if (condition2) btn = Y;
```